airflow/models/baseoperator.py

```diff
@@ -567,6 +567,13 @@ def __init__(
         self.email = email
         self.email_on_retry = email_on_retry
         self.email_on_failure = email_on_failure
+        self.execution_timeout = execution_timeout
+        self.on_execute_callback = on_execute_callback
+        self.on_failure_callback = on_failure_callback
+        self.on_success_callback = on_success_callback
+        self.on_retry_callback = on_retry_callback
+        self._pre_execute_hook = pre_execute
+        self._post_execute_hook = post_execute
```

Comment:
See comment in PR desc for why I've re-ordered these.

Reply:
I think this was the reason (self passed to logging which potentially reads all self fields):

```
self.log.warning("Implicitly converting 'retries' for %s from %r to int", self, retries)
```